core/templates/dev/head/domain/dashboard/DashboardBackendApiServiceSpec.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for DashboardBackendApiService.
+ */
+
+describe('Dashboard backend API service', function() {
+  var DashboardBackendApiService = null;
+  var sampleDataResults = null;
+  var $rootScope = null;
+  var $scope = null;
+  var $httpBackend = null;
+
+  beforeEach(module('oppia'));
+  beforeEach(module('oppia', GLOBALS.TRANSLATOR_PROVIDER_FOR_TESTS));
+
+  beforeEach(inject(function($injector) {
+    DashboardBackendApiService = $injector.get('DashboardBackendApiService');
+    $rootScope = $injector.get('$rootScope');
```

Comment:
Included accidentally. Removed in https://github.com/oppia/oppia/pull/2126/commits/43c3cde4e7ad12e026b77a1f3ebc00daa8722312